frontend/app/components/WelcomeScreen/animations/ScrollAnimation.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import { View, Text } from 'react-native';
+import Animated, { AnimatedStyle } from 'react-native-reanimated';
+import { Ionicons } from '@expo/vector-icons';
+import { useTheme } from '../../../contexts/ThemeContext';
+
+interface ScrollAnimationProps {
+  scrollStyles: AnimatedStyle<object>[];
+  pulseStyle: AnimatedStyle<object>;
+  colors: ReturnType<typeof useTheme>['colors'];
+}
+
+export const ScrollAnimation: React.FC<ScrollAnimationProps> = ({
+  scrollStyles,
+  pulseStyle,
+  colors,
+}) => {
+  const expressions = [
+    { emoji: '🌍', text: 'Around the world', meaning: 'En todo el mundo' },
+    { emoji: '🎨', text: 'Piece of cake', meaning: 'Pan comido' },
+    { emoji: '🚀', text: 'Shoot for stars', meaning: 'Apuntar alto' },
+  ];
```

Comment:
I think this does not really represent what the app does, we don't have translations yet, the meaning here is something different